libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/BuildCacheRelocationIT.kt

```diff
@@ -128,7 +128,8 @@ class BuildCacheRelocationIT : BaseGradleIT() {
                      cacheableTaskNames = listOf(
                          "kaptKotlin", "kaptGenerateStubsKotlin", "compileKotlin", "compileTestKotlin", "compileJava"
                      ),
-                     initProject = { File(projectDir, "build.gradle").appendText("\nkapt.useBuildCache = true") }
+                     initProject = { File(projectDir, "build.gradle").appendText("\nkapt.useBuildCache = true") },
+                     withAnotherGradleHome = true
```

Comment:
Hey, I don't know exactly what `withAnotherGradleHome` does but since the build cache is stored in the Gradle Home directory would that mean that we are not able to use the build cache for the relocation test? For a cache relocation test, we would want to use the same Gradle home but a different project directory.

Reply:
This test covers the test case in the commit message. The build cache location is the same for both builds, outside of gradle user home, and builds use different gradle user home to get different location for kotlin stdlib.